src/pages/blog/leaf-is-five-years-old.pug

```diff
@@ -0,0 +1,112 @@
+
+            p.blog-post__text We've still not won this battle, but don't need to be convinced of the benefit. More than half of our time is now spent working <em>on</em> Leaf, rather than <em>in</em> Leaf. Managing people, clients, sales funnels, and marketing strategies takes time, and isn't optional.
+
+            p.blog-post__text Build yourself a team capable of doing great work, and support them with the structure and an environment that lets them do just that.
+
+            h3.blog-post__h3 Delegate where possible
+
+            p.blog-post__text We're surrounded by talented, highly skilled individuals, who are able to take a portion of our workloads off our plate, but so often we don't feel comfortable delegating.
+
+            p.blog-post__text By taking ownership of every detail, you not only spread yourself too thinly, but you create unnecessary bottlenecks. Your skills may seen considerable, but like your time, they only go so far.
+
+            p.blog-post__text Learn to let go, and trust in others to not completely screw it up. They might even surprise you.
+
+            h3.blog-post__h3 Success isn't measured in pound signs
+
+            p.blog-post__text Money is of course important, but it is not, and will never be, how we measure success. When the number of £s in your company account is the main barometer for how well you're doing, you're setting yourself up to make rash, tired, and selfish decisions that negatively impact everyone around you.
+
+            p.blog-post__text Rather than fetishising making bank, passionately chase adding value; the more valuable you are, the more desirable your services, and the money you'll make regardless.
+
+            h3.blog-post__h3 Work smarter, not harder
+
+            p.blog-post__text The grind is not an obligation or a badge of honour, and if you're feeling the need to consistently work more than 40 hours a week, you're either not working effectively enough, or you're working for arseholes.
+
+            p.blog-post__text If you're lucky, your job will be something you're passionate about. It may well have been your hobby before you ever got paid to do it, or you may have stumbled into it somewhere along your journey. But either way, it's still a job.
+
+            p.blog-post__text When you work a long and gruelling schedule, you're not showing up for yourself, your family, or your friends. A single parent working 3 minimum-wage jobs to put dinner on the table for their kids is a selfless act; working into your evenings or weekends for a slave-driving client is not.
+
+            +permalink({
+              heading: "What's next"
+            })
+
+            p.blog-post__text As we mentioned earlier, we're hopeful that we'll be a team of six within the next couple of months with the addition of a new senior frontend developer.
```

Comment:
Not a big deal, but while it is a senior position, it's not marketed that way on the job ad. The only clue is the salary band.